### ISSUE TITLE
CI lint failure after removing Storage

### ISSUE THREAD

**Issue by stephenplusplus** [GoogleCloudPlatform/google-cloud-node#2628]
@callmehiphop any ideas?

https://circleci.com/gh/GoogleCloudPlatform/google-cloud-node/1823:

```
> google-cloud-node@ lint /Users/distiller/google-cloud-node
> jshint scripts/ packages/ system-test/ test/ && jscs packages/ system-test/ test/

git rev-parse --abbrev-ref HEAD
master
git diff HEAD^ --name-only
README.md
packages/storage/README.md
packages/storage/package.json
packages/storage/src/acl.js
packages/storage/src/bucket.js
packages/storage/src/channel.js
packages/storage/src/file.js
packages/storage/src/iam.js
packages/storage/src/index.js
packages/storage/system-test/data/CloudPlatform_128px_Retina.png
packages/storage/system-test/data/long-html-file.html
packages/storage/system-test/data/long-html-file.html.gz
packages/storage/system-test/data/three-mb-file.tif
packages/storage/system-test/storage.js
packages/storage/test/acl.js
packages/storage/test/bucket.js
packages/storage/test/channel.js
packages/storage/test/file.js
packages/storage/test/iam.js
packages/storage/test/index.js
packages/storage/test/testdata/privateKeyFile.json
packages/storage/test/testdata/testfile.json
packages/storage/test/testdata/textfile.txt

module.js:529
    throw err;
    ^

Error: Cannot find module '/Users/distiller/google-cloud-node/packages/storage/package.json'
    at Function.Module._resolveFilename (module.js:527:15)
    at Function.Module._load (module.js:476:23)
    at Module.require (module.js:568:17)
    at require (internal/module.js:11:18)
    at new Module (/Users/distiller/google-cloud-node/scripts/helpers.js:63:17)
    at Module (/Users/distiller/google-cloud-node/scripts/helpers.js:57:12)
    at Array.map (<anonymous>)
    at Function.Module.getUpdated (/Users/distiller/google-cloud-node/scripts/helpers.js:142:24)
    at Object.<anonymous> (/Users/distiller/google-cloud-node/scripts/build.js:50:22)
    at Module._compile (module.js:624:30)

. scripts/circle/test.sh 8 returned exit code 1
```

**Comment by callmehiphop** [comment]
The bulk of it comes from trying to see what files changed via [git diff](https://github.com/GoogleCloudPlatform/google-cloud-node/blob/master/scripts/helpers.js#L85). It reports back that storage has changed, but not that it has been deleted. We might want to alter that script to filter modules out that no longer exist since we'll probably run into the same problem as we extract each module.

**Comment by stephenplusplus** [comment]
Thanks!

**Comment by stephenplusplus** [comment]
Fixed by #2629.
